composer.json

```diff
@@ -53,6 +53,7 @@
     "require-dev": {
         "phpunit/phpunit": "^9.5",
         "symfony/browser-kit": "^5.4|^6.0",
+        "symfony/clock": "^6.3",
```

Comment:
as you instantiate a Symfony Clock when no clock is provided, this is not really an optional dependency.

Reply:
Thanks for the feedback.

I am not sure if this should be tied to "symfony/clock" or if it is possible to just add the requirement that it needs a repo that implements "psr/clock" and let the user decide, which one to chose. require [psr/clock-implementation](https://packagist.org/providers/psr/clock-implementation)?
https://packagist.org/?query=clock

The linked PR from @Dean151 makes the dependency optional.

Reply:
Well, your implementation is the one making it non-optional.

However, as this package is a Symfony bundle, I would not be too concerned about being bring a dependency on the Symfony component.